docs/cloud/integrations/grafana.md

```diff
@@ -0,0 +1,1011 @@
+
+With Grafana up and running, you can access it from the browser through `http://localhost:3000`. Click on `Data Sources` under the configuration option, then click `Add Data Source`, select `PostgreSQL` and enter the connection details. Since there is no field for port number, it should be added with the host as `Host:Port` and set the TLS/SSL mode to `Require`. Then click `Save & Test` to test your connection.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/grafana-connection-success.png" />
+</div>
+
+The plugins and it's tables can be accessed through the `Explore` tab and the `Edit SQL` option can be used to customize the queries.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/grafana-explore-tables.png" />
+</div>
+
+The data can be previewed in a `Table format` or `Time series`. Here we see the AWS EC2 instances listed in a table format.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/grafana-table-format-data-preview.png" />
+</div>
+
+## Dashborad to monitor resources and cost
+
+Grafana uses panels as the building blocks of a dashboard that consists of queries, graphs, aliases, etc. We'll focus here on creating a dashboard to monitor AWS services and Cost. To being click on `Dashboards` and create a new dashboard. On the empty dashboard click `Add panel` to create a panel and select `aws_ebs_volume_metric_read_ops_daily` from the query builder, then click `Apply`. Now add panels for `aws_ec2_instance_metric_cpu_utilization_hourly`, `aws_s3_bucket_by_region` and `aws_vpc_by_region`. Similarly paste the query into a new panel to monitor the top 10 monthly costs by service.
+
+```
+select
+  service,
+  sum(unblended_cost_amount)::numeric::money as sum,
+  avg(unblended_cost_amount)::numeric::money as average
+from
+  aws_cost_by_service_monthly
+group by
+  service
+order by
+  average desc
+limit 10;
+```
+Grafana gives a wide range of graphics that you can use to represent the data in each panel. Now with the created panels we have our AWS Monitoring dashboard ready.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/grafana-dashboard.png" />
+</div>
+
+## Import a dashboard with JSON file
+
+Since, a dashboard in Grafana is represented by a JSON object, which stores its metadata. We can [import a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/export-import/#import-dashboard) using json input. To import click `Import` under the Dashboards icon in the side menu and paste the below code to create a AWS S3 dashboard. With the Enterprise licensed feature you can send the dashboards as reports.
```

Comment:
```suggestion
A Grafana dashboard is represented by a JSON object that stores its metadata. Here is the JSON definition for an AWS S3 dashboard. 

[move details widget here]

[move image here]

To import it, click `Import` under the Dashboards icon in the side menu, then copy and paste the JSON.
```

Reply:
Nice use of the details widget. It's /so/ handy, eh?

Reply:
Yeah. Found this to be one of the best features to instantly share a Working dashboard with anyone.

Reply:
Made the requested changes